docs/app/partials/home.tmpl.html

```diff
@@ -289,4 +290,3 @@ <h2 class="md-headline">Conference Presentations:</h2>
     </p>
   </md-content>
 </div>
-
```

Comment:
The blank line at the end of the file should remain.

Reply:
This must have been IntelliIJ... I'll update. Thanks.